lua/harpeek/extensions/arrow.lua

```diff
@@ -9,7 +9,7 @@ return {
             pattern = 'ArrowUpdate',
             callback = function()
                 if Harpeek._window then
-                    Harpeek.open()
+                    Harpeek.open(Harpeek._open_opts)
```

Comment:
There's a few places where we're passing this into open. Do we need to? Inside of `open` we could just check if we have any "open opts" and if we do use them. That way we're only doing this in one place.

Reply:
I don't think we could go without this thing, because we can't differentiate between user call and callback call inside of open. But if i dont use Hapeek._open_opts, then user call without opts will use open_opts, instead of default settings.

Maybe i could add a wrapper func around open which we will call inside callback, that will pass open_opts?
But this approach may add unnecessary complexity

Reply:
Wrapper func would look like this
```lua
function Harpeek._update()
    Harpeek.open(Harpeek._open_opts)
end
```

Reply:
I see what you're saying. Yeah I much prefer this approach with the update wrapper. Just need to add a check that the window exists first so we don't forcibly open the window on update. 

Also we should probably nil out the _open_opts on window close.